contracts/.solcover.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  mocha: {
+    grep: "@skip-coverage", // Find everything with this tag
```

Comment:
Given `solidity-coverage` compiles the contract without the optimizer, we use this tag on any tests that rely on gas measurements.